File: python/数组/1085. 最小元素各数位之和.py
from typing import List
class Solution:
    def sumOfDigits(self, A: List[int]) -> int:
        min_num = min(A)
        ans = 0
        for i in str(min_num):
            ans += int(i)
        return 1 if ans % 2 == 0 else 0

File: go/数组/1085. 最小元素各数位之和.go
package main

import "math"

func sumOfDigits(A []int) int {
	var ans int
	minNum := math.MaxInt32
	for _, v := range A {
		if v < minNum {
			minNum = v
		}
	}
	for minNum != 0 {
		ans += minNum % 10
		minNum /= 10
	}
	if ans%2 == 0 {
		return 1
	} else {
		return 0
	}
}
